Formula/irssi.rb

```diff
@@ -44,7 +44,7 @@ def install
     end
 
     # https://github.com/irssi/irssi/pull/927
-    inreplace "configure", "^DUIfm", "^DUIifm"
+    # inreplace "configure", "^DUIfm", "^DUIifm"
```

Comment:
If we remove this inreplace we should take out all the comments related to it as well before merging.